Apps/WebClient/src/ClientApp/app/models/timelineEntry.ts

```diff
@@ -0,0 +1,25 @@
+import MedicationStatement from "@/models/medicationStatement";
+
+export enum EntryType {
+  Medication,
+  Laboratory,
+  NONE
+}
+
+// Timeline entry model
+export default class TimelineEntry {
+  public readonly date: Date;
+  public readonly id: string;
+  public readonly title: string;
+  public readonly description: string;
+  public readonly type: EntryType;
+
+  public constructor(other: MedicationStatement) {
+    // The ID needs to come from the the server
+    this.id = "id-" + Math.random();
```

Comment:
lets move this to the medication service?

Reply:
Correct. However doing that will trickle all the way down to the parsing of the trf message. Probably another task